### PR TITLE
Publishing paper to JOSS

### PR DESCRIPTION
**Pull request by arunavabasu-03**
Publishing paper to JOSS

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
+aas-journal: Astrophysical Journal <- The name of the AAS journal.
+---
+# Summary
+
+**Radis App** is a web application for Radis high-resolution infrared molecular spectra. Instead of writing code, this project aims to create an intuitive user interface (UI).
+It use Radis internally to produce spectrum, and the updated version and Radis algorithm make it incredibly efficient to compute the millions of lines in only a few minutes.
```
Comment (suzil) [inline paper/paper.md:28]:
```suggestion
It uses Radis internally to produce spectrum, and the updated version and Radis algorithm makes it incredibly efficient to compute millions of lines in only a few minutes.
```

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Radis app leverages React 18 to offer the user interface, and FastApi on the backend. We are using react-hook-form for the fastest user experience and to maintain performance on the client slide. In the backend, we use FastApi to offer the fastest response.
+
+# Statement of need
+
+We build  **Radis App** to  address an important need in the field of high-resolution infrared molecular spectroscopy. Traditional approaches for analyzing  molecular spectra often involve complex coding and lack intuitive user interfaces, leading to barriers for researchers and practitioners in effectively utilizing this valuable data.
```
Comment (suzil) [inline paper/paper.md:33]:
```suggestion
We build  **Radis App** to address an important need in the field of high-resolution infrared molecular spectroscopy. Traditional approaches for analyzing  molecular spectra often involve complex coding and lack intuitive user interfaces, leading to barriers for researchers and practitioners in effectively utilizing this valuable data.
```

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+The Radis app fills this gap by providing a user-friendly web application that leverages the power of the Radis algorithm. By abstracting  the need for coding, the Radis app enables researchers to efficiently generate high-resolution spectra with ease. This not only saves significant time and effort but also opens up opportunities for a wider range of users to explore and analyze molecular spectra.
```
Comment (suzil) [inline paper/paper.md:35]:
```suggestion
The Radis app fills this gap by providing a user-friendly web application that leverages the power of the Radis algorithm. By abstracting the need for coding, the Radis app enables researchers to efficiently generate high-resolution spectra with ease. This not only saves significant time and effort but also opens up opportunities for a wider range of users to explore and analyze molecular spectra.
```

File: paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- **Radis App Architecture -**
+
+![Radis App Architecture](./images/radisapp_architecture.png)
+
+
+# Features
+
+The Radis app offers several key features that enhance the user experience and facilitate efficient analysis of high-resolution infrared molecular spectra. These features includes -
+
+1. **Plotting Spectra**: The app allows users to plot the generated spectra visually. This feature provides a clear and concise representation of the spectral data, enabling researchers to easily analyses and interpret the results. Visualizations can be customized and adjusted to meet specific requirements, enhancing data comprehension and facilitating further analysis.
```
Comment (suzil) [inline paper/paper.md:54]:
```suggestion
1. **Plotting Spectra**: The app allows users to plot the generated spectra visually. This feature provides a clear and concise representation of the spectral data, enabling researchers to easily analyze and interpret the results. Visualizations can be customized and adjusted to meet specific requirements, enhancing data comprehension and facilitating further analysis.
```

**Comment by arunavabasu-03** [comment]
@suzil i build the draft but the reference is not showing, can you check pls 😅!!

**Comment by arunavabasu-03** [comment]
@suzil we can merge For now !! Lets see their feedback !!

**Comment by suzil** [comment]
> @suzil we can merge For now !! Let's see their feedback !!

Could the failing build be fixed?

**Comment by arunavabasu-03** [comment]
And alos setup codecov for the backend , because it needs some additional setup to show seperately the backend and frontend !!
Screenshot 2023-07-19 at 9 05 13 PM

**Review by suzil — APPROVED** [review]

**Pull request merged**